src/dftbp/extlibs/tblite.F90

```diff
@@ -1029,10 +1186,54 @@ end subroutine buildDiatomicBlocks
 #:endif
 
 
+  !> Shift multipole operator from Bra function (center i) to Ket function (center j)
```

Comment:
Closest is probably a horizontal Obara–Saika moment shift (see https://aip.scitation.org/doi/10.1063/1.450106).

Reply:
Could you add a comment pointing towards this somewhere?

Reply:
I think it is even simpler than integral calculations, just standard multipole displacement rules. I added some comments on how this is done here, since the trace removal for the quadrupoles is not directly obvious on the first glance.